.github/dependabot.yml

```diff
@@ -2,16 +2,9 @@
 version: 2
 updates:
   - package-ecosystem: "gomod"
-    directory: "/"
-    schedule:
-      interval: "daily"
-    labels:
-      - automation
-    reviewers:
-      - "elastic/ecosystem"
-    open-pull-requests-limit: 10
-  - package-ecosystem: "gomod"
-    directory: "/compliance"
+    directories:
+      - "/"
+      - "/compliance"
```

Comment:
Trying to group dependabot updates in once, to try to avoid issues with compliance directory go.mod being out of sync with the root go.mod.